src/rpc/misc.cpp

```diff
@@ -497,14 +497,18 @@ UniValue setmocktime(const UniValue& params, bool fHelp)
     if (fHelp || params.size() != 1)
         throw runtime_error(
             "setmocktime timestamp\n"
-            "\nSet the local time to given timestamp (-regtest only)\n"
+            "\nSet the local time to given timestamp (-regtest only).\n"
+            "The node must be started with `-mocktime` in order to use this API.\n"
             "\nArguments:\n"
             "1. timestamp  (integer, required) Unix seconds-since-epoch timestamp\n"
             "   Pass 0 to go back to using the system time."
```

Comment:
This is no longer true:
```suggestion
            "1. timestamp  (integer, required) Unix seconds-since-epoch timestamp."
```